src/hivpy/circumcision_data.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
+import yaml
+
+from hivpy.exceptions import DataLoadException
+
+from .common import DiscreteChoice
+
+
+class CircumcisionData:
+    """
+    Class to hold and interpret circumcision data loaded from the yaml file.
+    """
+
+    # TODO: This is ripped directly from sex_behaviour_data.py,
+    # we should make a new data reader module to store functions like this.
```

Comment:
Agreed! We need to remember to make an issue for this. It might make sense to have a data class, which all the specific data classes inherit from, that has these methods